common/src/main/java/com/wynntils/mc/render/FontRenderer.java

```diff
@@ -89,29 +126,46 @@ public void renderAlignedTextInBox(
             float y,
             float maxWidth,
             CustomColor customColor,
-            TextAlignment textAlignment,
+            HorizontalAlignment horizontalAlignment,
             TextShadow textShadow) {
-        switch (textAlignment) {
-            case LEFT_ALIGNED -> renderText(poseStack, text, x1, y, maxWidth, customColor, textAlignment, textShadow);
-            case CENTER_ALIGNED -> renderText(
-                    poseStack,
-                    text,
-                    x1 + (x2 - x1 - font.width(text)) / 2,
-                    y,
-                    maxWidth,
-                    customColor,
-                    TextAlignment.LEFT_ALIGNED,
-                    textShadow);
-            case RIGHT_ALIGNED -> renderText(
-                    poseStack,
-                    text,
-                    x2 - font.width(text),
-                    y,
-                    maxWidth,
-                    customColor,
-                    TextAlignment.LEFT_ALIGNED,
-                    textShadow);
-        }
+
```

Comment:
Spotless is not really helpful here.

But, there's an extra empty line which just makes a bad line look even worse. Can you remove it? (also in the next method)